SpatialGDK/Source/Private/Interop/SpatialSender.cpp

```diff
@@ -18,6 +20,7 @@
 void USpatialSender::Init(USpatialNetDriver* InNetDriver)
 {
 	NetDriver = InNetDriver;
+	View = InNetDriver->View;
```

Comment:
Could this be converted to an initializer list?

Reply:
This isn't a constructor.

Reply:
I have 0 brain cells.